recipe/meta.yaml

```
@@ -1,24 +1,26 @@
-{% set version = "1.0.2" %}
+{% set name = "sphinxcontrib-devhelp" %}
+{% set version = "2.0.0" %}
 
 package:
-  name: sphinxcontrib-devhelp
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/s/sphinxcontrib-devhelp/sphinxcontrib-devhelp-{{ version }}.tar.gz
-  sha256: ff7f1afa7b9642e7060379360a67e9c41e8f3121f2ce9164266f61b9f4b338e4
+  url: https://pypi.org/packages/source/s/sphinxcontrib-devhelp/sphinxcontrib_devhelp-{{ version }}.tar.gz
+  sha256: 411f5d96d445d1d73bb5d52133377b4248ec79db5c793ce7dbe59e074b4dd1ad
 
 build:
   number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv --no-build-isolation
   noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
-    - python >=3.5
+    - python 3.9
     - pip
+    - flit-core >=3.7
   run:
-    - python >=3.5
+    - python 3.9
```

```suggestion
    - python >=3.9
```
Should be `>=` in run